routes/v1.js

```diff
@@ -1040,18 +1063,19 @@ api.declare({
   // Construct status object
   var status = task.status();
 
-  // Publish message about last run, if it was canceled
+  // If the last run was canceled, resolve dependencies and publish message
   if (run.state === 'exception' && run.reasonResolved === 'canceled') {
```

Comment:
he he, correctness is enforced by JSON schema for input and output...

Having an enum-object-like-thing like `reasons.CANCELLED` whose value is the string  would look nice.
But if we referenced a property that doesn't exist we would get `undefined` which is a value different from `run.reasonResolved`.

So I'm basically not sure how to do enums better... At least this way it's clear that it's fragile...